gammapy/time/exptest.py

```diff
@@ -0,0 +1,45 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import numpy as np
+
+__all__ = [
+    'exptest_for_run',
+]
+
+def exptest_for_run(time_delta=[]):
+    """Compute Mr value, the level of variability for a certain period of time.
+
+    Longer description: A single Mr value can be calculated, which shows the level of variability
```

Comment:
Remove the text "Longer description:" here.
I.e. keep the description, but don't preface it with "Longer description:"